dashboard/app/views/scripts/show.html.haml

```diff
@@ -66,7 +66,7 @@
 - if current_user && current_user.permission?(UserPermission::LEVELBUILDER)
   -# Show all the levels, their names, and instructions in the extra links box.
   = render layout: 'shared/extra_links' do
-    - levels = Script.get_from_cache(@script.name).stages.map{ |stage| stage.script_levels.map{ |sl| sl.level }}.flatten
+    - levels = Script.get_from_cache(@script.name).script_levels.map{ |sl| sl.levels }.flatten
```

Comment:
Not sure, but https://github.com/code-dot-org/code-dot-org/pull/10928 made this change unnecessary anyway. Reverted except for changing sl.level to sl.levels to include any level variants.